pypeit/calibrations.py

```diff
@@ -774,7 +774,13 @@ def get_wv_calib(self):
         # Grab arc binning (may be different from science!)
         arc_rows = self.fitstbl.find_frames('arc', calib_ID=self.calib_ID, index=True)
         self.arc_files = self.fitstbl.frame_paths(arc_rows)
-        binspec, binspat = parse.parse_binning(self.spectrograph.get_meta_value(self.arc_files[0],
+        if len(self.arc_files) == 0:  # Better be reading from Masters
+            if not self.reuse_masters:
+                msgs.error("At least one arc file is required!")
+            binspec, binspat = 1,1
```

Comment:
Same as above.

Reply:
Not resolved until you demonstrate why these lines of code are needed. I think you failed to set something related to master files, since the code cannot run without an arc.

Reply:
Reverted